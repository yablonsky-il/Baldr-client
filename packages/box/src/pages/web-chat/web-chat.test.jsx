import WebChat from './web-chat';

describe('<WebChatUI />', () => {
  const wrapper = shallow(<WebChat />);

  test('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
