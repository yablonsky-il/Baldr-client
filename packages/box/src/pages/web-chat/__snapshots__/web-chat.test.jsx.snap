// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WebChatUI /> should match snapshot 1`] = `
<div>
  Section in development
</div>
`;
